src/test/java/org/formflowstartertemplate/app/journeys/LaDocUploadFlowJourneyTest.java

```diff
@@ -27,13 +27,14 @@ void fullUploadDocumentFlow() {
 
     // Upload documents
     assertThat(testPage.getTitle()).isEqualTo("Upload documents");
-    assertFalse(testPage.findElementById("form-submit-button").isEnabled());
+    assertThat(testPage.findElementById("form-submit-button").getAttribute("class").contains("disabled")).isTrue();
```

Comment:
I made a corresponding change to FFL to use the disabled property. I _think_ that's better for accessibility but we can double check - https://github.com/codeforamerica/form-flow/pull/138

Reply:
platform teams is working on this